### ISSUE TITLE
postShare() method acutally haven't set the title

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#51]
```
What steps will reproduce the problem?
1. generate a LinkedInApiClient with consumer and access token/secret
2. try to post a new share,with comment,title,url,etc.

What is the expected output? What do you see instead?
The output should have set the title for the new share,but no.When check the 
source code(BaseLinkedInApiClient.class) i find that the title is not set.

What version of the product are you using? On what operating system?
linkedin-j-1.0.361.zip

Please provide any additional information below.


```

Original issue reported on code.google.com by `se...@163.com` on 24 Mar 2011 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
Thanks for pointing this out. I will fix this in the next build over the 
weekend.
```

Original comment by `nabeelmukhtar` on 24 Mar 2011 at 7:59
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
This issue is fixed in build 1.0.367.
```

Original comment by `nabeelmukhtar` on 5 Apr 2011 at 12:59
- Changed state: **Fixed**
